packages/xo-server/src/xo-mixins/backups-remote-adapter.js

```diff
@@ -0,0 +1,27 @@
+import Disposable from 'promise-toolbox/Disposable'
+
+import { compose } from '@vates/compose'
+import { decorateWith } from '@vates/decorate-with'
+import { deduped } from '@vates/disposable/deduped'
+import { RemoteAdapter } from '@xen-orchestra/backups/RemoteAdapter'
+
+export default class BackupsRemoteAdapter {
```

Comment:
Why not on the `BackupsNg` mixin?

Reply:
Used also in the file restore.

Reply:
Ok, it's an implementation detail anyway :slightly_smiling_face: